src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -153,7 +153,13 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     const filename = this.props.frame.filename;
     const platform = this.props.event.platform;
 
-    if (this.project && this.integrations.length > 0 && filename) {
+    // filter out self hosted GitLab integrations from getting
+    // the CTA for now
+    const integrations = this.integrations.filter(
```

Comment:
@MeredithAnya I think this code belongs on the backend since we already have code that is filtering integrations. Better to keep all that logic in one place.

Reply:
Here: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_stacktrace_link.py#L49-L52